.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I don't think we want to check in IDE specific files. To my understanding, they should be ignored locally through the `.git/info/exclude` file.

Reply:
This is a rebase fail....fixing :(